samples/deployment/test-network/installFPC.sh

```diff
@@ -7,161 +7,43 @@
 
 set -euo pipefail
 
-#DEBUG=true # uncomment (or define when calling script) to show debug output
-# Note: to see only log and not peer command output (which is all on stderr),
-#       run the script with '... 2>/dev/null' ..
+CHANNEL_NAME=${CHANNEL_NAME:-mychannel}
+FABRIC_SAMPLES=${FPC_PATH}/samples/deployment/test-network/fabric-samples
```

Comment:
This does not look right. 
Isn't the fabric-samples module removed from this PR, and placed in a temp folder?

Reply:
no, it is still cloned into the test-network directory

Reply:
no, the tutorial uses the setup script to clone the fabric samples on demand. The integration test just does a spatial clone to get a single chaincode. While github allows to download individual files via curl, they don't allow to download entire folders.